iidm/iidm-xml-converter/src/main/resources/xsd/slackTerminal.xsd

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2020, RTE (http://www.rte-france.com)
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-->
+<xs:schema version="1.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:iidm="http://www.powsybl.org/schema/iidm/1_3"
+           targetNamespace="http://www.powsybl.org/schema/iidm/ext/slack_terminal/1_0"
+           elementFormDefault="qualified">
+    <xs:import namespace="http://www.powsybl.org/schema/iidm/1_3" schemaLocation="iidm_V1_3.xsd"/>
+    <xs:element name="slackTerminal">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="terminalRef" type="iidm:TerminalRef"/>
```

Comment:
Set the min/max occurs: this element is required

Reply:
Attributes are now directly to the slackTerminal object.